site/cds_rdm/inspire_harvester/transformer.py

```diff
@@ -21,7 +22,10 @@ def __init__(self, root_element=None, *args, **kwargs):
 
     def apply(self, stream_entry, **kwargs):
         """Applies the transformation to the INSPIRE record entry."""
+        current_app.logger.info("Start transformation of INSPIRE record to CDS record.")
         rdm_entry, errors = RDMEntry(stream_entry.entry).build()
+        current_app.logger.info(f"Transformed CDS entry: {rdm_entry}.")
+        current_app.logger.info(f"Errors: {errors}.")
         stream_entry.errors.extend(errors)
```

Comment:
here would be possibly the time to summarise all the transformation errors at once